puppet/modules/web/templates/yum/HEADER.html.epp

```diff
@@ -1,11 +1,12 @@
+<%- | String $stable | -%>
 <h1>yum.theforeman.org</h1>
 
 <h3>Foreman</h3>
 
 <p>Foreman is available under <code>/releases/VERSION/DIST/ARCH</code>, e.g.</p>
 
 <ul>
-  <li>/releases/3.8/el8/x86_64</li>
+  <li>/releases/<%= $stable %>/el8/x86_64</li>
   <li>/releases/nightly/el8/x86_64</li>
```

Comment:
Should we also encourage el9?
```suggestion
  <li>/releases/<%= $stable %>/el9/x86_64</li>
  <li>/releases/nightly/el9/x86_64</li>
```

Reply:
yes, did a `s/el8/el9/g` on the whole file